Sources/SMTP.swift

```diff
@@ -49,7 +49,7 @@ public struct SMTP {
     private let authMethods: [AuthMethod]
     private let domainName: String
     private let accessToken: String?
-    private let timeout: UInt
+    private let timeout: Int
```

Comment:
It looks like this PR was opened against [v1.1.0](https://github.com/IBM-Swift/Swift-SMTP/releases/tag/v1.1.0). Can you rebase against [v1.1.1](https://github.com/IBM-Swift/Swift-SMTP/releases/tag/v1.1.1)? This should fix these discrepancies.